com.ultimaker.cura.appdata.xml

```diff
@@ -25,9 +25,10 @@
   </description>
   <screenshots>
     <screenshot type="default">
-      <image>https://raw.githubusercontent.com/Ultimaker/Cura/master/screenshot.png</image>
+      <image>https://raw.githubusercontent.com/Ultimaker/Cura/4.13.1/cura-logo.PNG</image>
```

Comment:
Best not make this image refer to a specific branch. We wouldn't want to update it every release.